const mongoose = require('mongoose');

function connectDabatase() {
const url = 'mongodb+srv://sarkis:ssffdd66@cluster0-cbwpb.mongodb.net/test?retryWrites=true&w=majority';

const options = {reconnectTries : Number.MAX_VALUE,reconnectInterval : 500,useNewUrlParser : true};

mongoose.connect(url,options);

mongoose.set('useCreateIndex',true);

mongoose.connection.on('error',(err) => {
    console.log('Erro na conex達o com o banco')
})

mongoose.connection.on('connected',(err) => {
    console.log('Sucesso na conex達o com o banco')
})

mongoose.connection.on('disconnected',(err) => {
    console.log('Perda de conex達o com o banco')
})
}

exports.connectDabatase = connectDabatase; // usando module.exports n達o funciona para este caso